bldr/data/hooks/pre-install-deps

```diff
@@ -1 +1,27 @@
 #!/bin/bash
+
+perform_unminimization() {
+    unminimize_cmd=$(command -v unminimize)
+    local unminimize_present=$?
+
+    if [[ ${unminimize_present} == 0 ]]; then
```

Comment:
Or drop this variable and use [[ "${unminimize_cmd}" ]] here.

Reply:
the current version is more explicit